sanitizers/sql.go

```diff
@@ -0,0 +1,94 @@
+package sanitizers
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"regexp"
+)
+
+var syntaxErrors = []*regexp.Regexp{
+	regexp.MustCompile(`\S+ ERROR 1064 \(42000\): You have an error in your SQL syntax.*`), // MySQL error message
+	regexp.MustCompile(`\S+ ERROR: syntax error at or near .* \(SQLSTATE 42601\)`),         // PostgreSQL error message
+}
+
+func ExecDB(fakPC int, db *sql.DB, query string, args ...any) (sql.Result, error) {
+	result, err := db.Exec(query, args...)
+	if isSyntaxError(err) {
```

Comment:
Guide the fuzzer towards an invalid syntax? E.g. escape characters which would break the query.

Reply:
Done. Also added hooks for the functions that do not return an error to guide the fuzzer to produce errors in the corresponding functions returning errors